plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseTestElasticsearchConnectorTest.java

```diff
@@ -106,6 +155,68 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    @Override
+    public void testPredicateReflectedInExplain()
+    {
+        // The format of the string representation of what gets shown in the table scan is connector-specific
+        // and there's no requirement that the conform to a specific shape or contain certain keywords.
+
+        assertExplain(
+                "EXPLAIN SELECT name FROM nation WHERE nationkey = 42",
+                "nationkey::bigint", "::\\s\\[\\[42\\]\\]");
+    }
+
+    @Test
+    @Override
+    public void testSortItemsReflectedInExplain()
+    {
+        // The format of the string representation of what gets shown in the table scan is connector-specific
+        // and there's no requirement that the conform to a specific shape or contain certain keywords.
+        assertExplain(
+                "EXPLAIN SELECT name FROM nation ORDER BY nationkey DESC NULLS LAST LIMIT 5",
+                "TopNPartial\\[5 by \\(nationkey DESC");
+    }
+
+    @Test
+    @Override
+    public void testLimitPushDown()
```

Comment:
I strongly feel about renaming the test in ATDQ to `testLimitQueries` - it doesn't test pushdown at all and most subclasses already have another test called `testLimitPushdown` (vs `testLimitPushDown`) which is very confusing/error-prone. cc: @losipiuk
I can do this as a part of https://github.com/trinodb/trino/pull/7495.

No change requested here.

Reply:
Should we just merge with `testLimit`?